dune_client/client.py

```diff
@@ -128,9 +129,25 @@ def get_result_csv(self, job_id: str) -> ExecutionResultCSV:
         response.raise_for_status()
         return ExecutionResultCSV(data=BytesIO(response.content))
 
-    def get_latest_result(self, query: Query) -> ResultsResponse:
-        """GET latest results from Dune API for `query_id`"""
-        response_json = self._get(route=f"query/{query.query_id}/results")
+    def get_latest_result(self, query: Union[Query, str, int]) -> ResultsResponse:
+        """
+        GET the latest results for a query_id without having to execute the query again.
+
+        https://dune.com/docs/api/api-reference/latest_results/
+        """
+        if isinstance(query, Query):
+            params = {
+                f"params.{p.key}": p.to_dict()["value"] for p in query.parameters()
+            }
```

Comment:
interesting.... is this a new style?

Reply:
yes, i was surprised too :)

https://dune.com/docs/api/api-reference/latest_results/#python

<img width="690" alt="image" src="https://github.com/cowprotocol/dune-client/assets/40689054/3b01c1c8-882e-4b6b-bbaa-d8d5f81b0872">

Reply:
Maybe you can add some tests for this route.

Reply:
Yes, I've tested that locally. It's working.

I'll try to add some tests